DEVELOPER.md

```diff
@@ -39,6 +39,28 @@ We have four different groups of commands withing the Makefile
 
 Please run the testing targets locally before you push a PR.
 
+## Running Locally
+
+We use [kind] to run the operator in a local environment. This can provide a faster feedback cycle while developing since
+there's no need to set up a remote GKE/OpenShift cluster.
+
+[kind]: https://kind.sigs.k8s.io/
+
+**make dev/run**
+
+This command will get everything set up for you to begin testing out the operator. Specifically it will:
+
+* Start a kind cluster named test (context=kind-test)
+* Create the `dev` namespace
```

Comment:
Why `dev` don't our examples assume `default` namespace?

Reply:
No particular reason, just a habit of mine. Think we should just use _default_ ?

Reply:
My assumption would be that I could just copy paste the example and have it work and this breaks that assumption IMO. 

Where would you be using this?